### ISSUE TITLE
no result for "Install Shell Command"

### ISSUE THREAD

**Issue by DanielKehoe** [lektor/lektor#417]
I just downloaded the Lektor MacOS app. I selected "Install Shell Command" and confirmed "Yes" but nothing happens. I opened the Terminal and I get "lektor: command not found" in my console:
```
$ lektor
-bash: lektor: command not found
$ python --version
Python 2.7.6
$ python3  --version
Python 3.6.2
```
I'm not an experienced Python user so I don't know if I have Python set up correctly for Lektor.

Any suggestions about how to troubleshoot? Should I be able to use Lektor from the command line as soon as I run "Install Shell Command" from the application?

**Comment by wgwz** [comment]
There shouldn't be anything extra to do on the python side to get this working. One thing you could try if you haven't already is opening a new terminal, perhaps you tried in a stale terminal? Did you follow from [here](https://www.getlektor.com/docs/installation/)?

**Comment by DanielKehoe** [comment]
I tried with a new terminal and got `lektor: command not found`. I went ahead and installed from the command line and I was able to build a new project and open it in the app.

I'm fine now but you want to test this further to ensure a good initial user experience.

**Comment by nixjdm** [comment]
Thanks, I'll see if I can reproduce this in a few days. If anyone else can reproduce please report in :) Sometimes these Mac installer issues are a bit of a pain, but if I can reliably reproduce it that'll help a bunch.

**Comment by nixjdm** [comment]
I've reproduced your problem, and found someone else experiencing it https://stackoverflow.com/questions/45199844/lektor-os-x-desktop-app-install-shell-command-fails-when-anaconda-is-installed, though I'm not sure it's actually related to conda. For now @DanielKehoe if you need the command line tools, you'll need to instal via the curl command, like you have.

**Comment by theconsultant** [comment]
I'm seeing the same problem.. I do not have conda installed. If I could make heads-or-tails of the new way Console works in Sierra I might be able to give more useful info! ;-)

I can delay installing the CLI tool for a few days if anyone has tests/ideas of things for me to try, look for, etc.

**Comment by nixjdm** [comment]
@theconsultant From the SO article I linked to, that guy's solution is to just install lektor via the command line (in his case, using conda), to provide terminal support. The Desktop App (using the dmg) can still provide the gui, but using it to run a command line lektor is broken.

**Comment by theconsultant** [comment]
Thanks, @nixjdm I understood that.. i'm offering to wait and try alternative fixes, or tests to nail down what is causing the issue, if you have ideas or suggestions...

**Comment by nixjdm** [comment]
Ok, thanks @theconsultant, just wanted to make sure. 

Looking in the search https://github.com/lektor/lektor/search?p=1&q=install+shell+command&type=&utf8=%E2%9C%93 it looks like one of these is failing somehow:
https://github.com/lektor/lektor/blob/master/gui/static/js/lektorInterop.jsx#L61
https://github.com/lektor/lektor/blob/master/gui/resources/lektor-mac-proxy#L25

**Comment by theconsultant** [comment]
Is it possible the Mac version is not making it all the way to the install section? If I do `ln -fs /Applications/Lektor.app/Contents/Resources/local/bin/lektor-proxy /usr/local/bin/lektor` it works fine, and just for good measure I checked `mdfind "kMDItemCFBundleIdentifier == 'org.pocoo.lektor'" | head -1 | xargs -0 dirname` and it works fine here too. 

Either way the simplest fix for Mac users of Lektor who have this same issue is to run the symlink command, assuming they have the Lektor app in /Applications and are an admin on their own machine:
`ln -fs /Applications/Lektor.app/Contents/Resources/local/bin/lektor-proxy /usr/local/bin/lektor`

I then used the command line `lektor quickstart` to create a new lektor project and made changes using the Lektor.app. 👍
